account_payment_partner/tests/test_account_payment_partner.py

```diff
@@ -391,12 +396,25 @@ def test_partner_onchange(self):
         self.assertEqual(
             self.supplier_invoice.invoice_partner_bank_id, self.supplier_bank
         )
+        vals = {"partner_id": self.customer.id, "type": "out_refund"}
+        invoice = self.move_model.new(vals)
+        self.assertEquals(invoice.payment_mode_id, self.supplier_payment_mode)
+        vals = {"partner_id": self.supplier.id, "type": "in_refund"}
+        invoice = self.move_model.new(vals)
+        self.assertEquals(invoice.payment_mode_id, self.customer_payment_mode)
+
```

Comment:
Don't add empty lines. If you want to split things, put a comment and also explain the "section".